### DISCUSSION TITLE
我大抵是想她了……

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#5468] [灌水]
仿写的匆匆，你们多读几遍匆匆，肯定写的比我nb

我初中的时候，曾经有一位女同桌，那时，我们不过十三四岁，少不更事，心中隐隐约约有些对世界的疑惑和憧憬。

她说话不多，平时也显得很沉静，仿佛一棵生长在操场边的树，不引人注意，也不惹人厌烦。只是偶尔会在做作业时用余光瞟我一眼，目光中带着些许的轻蔑与好奇。

可是，有一日，这平静的湖面上突然投下一颗石子，激起了圈圈涟漪。那天放学后，她忽然神色凝重地对我说道：“你知道H是什么吗？”

我那时装作全然不知。“不知道，”我故作镇定地回道。她笑了，笑得有些神秘，就像是一位老练的赌徒手中握着一张未出示的王牌。

她开始讲了，讲一些我在课堂上从未听过的事情，那些似是而非的词汇在她嘴里流淌出来，像是黑暗中不知名的毒草。她说得兴致盎然，仿佛这些都是寻常事，而我，只是一旁的听众，似懂非懂，仿佛站在悬崖边，向下望去，满是迷雾。

“你听说过吗？这些都是人们之间的秘密。”她的声音低了下来，带着某种引诱与试探。我却感到一种莫名的压抑，心中有些惶恐，仿佛我们所谈的这些，不该属于我们年纪的人所知道。

那一刻，我忽然明白了，许多事情并不像我以前以为的那样单纯。她不是她，而我也不是我。世界，仿佛突然间变得陌生起来。而那些曾被我视作天真无邪的课堂和操场，也像是蒙上了一层阴影。

我们依旧做着同桌，依旧在课本上画着无关紧要的符号，依旧偶尔讨论一些并不重要的题目。但那次对话后，我总觉得有些东西变了。她的眼神开始多了一种我说不清的东西，仿佛在考验我，又仿佛在引领我走入一条岔路。而我，站在路口，迟疑着，徘徊着，不知该往何处走。

后来，一场考试，让我们分开了。她离开时，背影显得格外轻快，仿佛从未承载过这沉重的世界。我还在原地，听着周围人们的笑声和喧嚣，突然感到一种从未有过的空虚。

我时常想起她，也时常问自己，那时的对话究竟意味着什么。那是一个未解的谜团，像是童年里一场不曾结束的梦，忽明忽暗。

我不知道他们给了我多少个日子；但那段初中的时光，仿佛已经遥远得不成样子，像是穿过了无数层记忆的迷雾，模糊不清。我坐在书桌前，偶尔翻开一本旧课本，书页的边角已被时光磨得泛黄，像是被无数双手轻轻掠过。我脑中忽然浮现出那个教室，那个充满了粉笔灰和青春气息的地方。那时的我还很稚嫩，似懂非懂地看着周围的一切，仿佛这些日子会永远延续下去。

在默默里算着，我才发觉，那段时光已经离我远去，仿佛是八千多个日子中的一小片云，悄无声息地飘过。而那些曾陪伴我的人、事，渐渐在记忆中变得模糊，仿佛正被一场细雨打湿，洇开成难以辨认的水渍。阿萍的面孔也依稀浮现在眼前，她的笑容、她那些带着一丝挑衅的话语，竟也随时间而褪色。我分不清，那究竟是我们真切经历过的片段，还是岁月在我心头留下的幻象。

初中三年的时光，就像一条无声的河流，从我的指缝中滑过。我曾以为那时的一切是多么真实、多么沉重，但如今回想起来，却只剩下淡淡的虚无感。那些我以为重要的课本、考试、还有那些短暂而激烈的友谊，在岁月的冲刷下，竟都成了记忆的尘埃，轻轻一吹，便散得无影无踪。

每当我试图抓住那时的某个瞬间，我的手却愈加空虚。那些我们曾认为无比漫长的课间、无所事事的夏日午后，还有偶尔从同桌口中溜出来的那几句“成人”话题，仿佛都变成了时间的碎片，融入了无尽的流逝中，再也无法重现。

我不禁头涔涔而泪潸潸了。那时的我，也许未曾意识到自己正在走过青春的河畔，而如今，当我站在时间的对岸回望时，河水已然远去，再也触碰不到那些曾经属于我的、真实而清晰的日子，而她也离我远去了……

https://github.com/user-attachments/assets/2386789c-db36-4a89-a56a-2e488146470a

https://github.com/user-attachments/assets/1ff87c8c-bb2c-4d54-adab-e44c6d4b6a51

**Comment by KELERSAMA** [comment]
bruh在经历4个章抽象高中生活和女班长与女班长闺蜜的打压下 终于开始寻找真我了 
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by 20231013** [comment]
有一段很有意思
她在破除我对她的幻象的同时，也间接地破除着我幻想的有着唯一单纯的世界
因为我装作不懂，为追上幻想而逃避现实，即便幻想被虚无冲散也不愿意拥抱现实，这就直接破除着我幻想的世界，其源头却是那没能切合现实的幻象
我想她了估计也只是对自己的青春纯然无力的嘲弄，或者也是感慨那个幻象创造世界的自己已经离我远去了……

**Comment by 323157364** [comment]
大佬好文采！！

**Reply by jsishsjsj** [reply]
仿写的匆匆，你多读几遍匆匆，肯定写的比我nb

**Comment by KZ7-JPG** [comment]
最希望看见疯狂星期四的一集

**Comment by zero-2526** [comment]
一眼KFC（划到底），怎么不是（划回去）

**Comment by boziqwq** [comment]
给你500丁尼继续写🤓

**Reply by jsishsjsj** [reply]
仿写的匆匆，你多读几遍匆匆，肯定写的比我nb
